FundParser/BL/Exceptions/ApiErrorException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FundParser.BL.Exceptions
```

Comment:
As per best practices, we should include at least 3 ctors: https://learn.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions#custom-exception-types